contrib/init/debian/gitea

```diff
@@ -14,17 +14,17 @@
 # Do NOT "set -e"
 
 # PATH should only include /usr/* if it runs after the mountnfs.sh script
-PATH=/sbin:/usr/sbin:/bin:/usr/bin
-DESC="Git with a cup of tea"
+PATH=/sbin:/usr/sbin:/bin:/usr/bin:/usr/local/bin
+DESC="Gitea - Git with a cup of tea"
 NAME=gitea
 SERVICEVERBOSE=yes
 PIDFILE=/var/run/$NAME.pid
 SCRIPTNAME=/etc/init.d/$NAME
-WORKINGDIR=/home/git/gitea
-DAEMON=$WORKINGDIR/$NAME
-DAEMON_ARGS="web"
+WORKINGDIR=/var/lib/$NAME
+DAEMON=/usr/local/bin/$NAME
+DAEMON_ARGS="web -c /etc/$NAME/app.ini"
 USER=git
-USERBIND="setcap cap_net_bind_service=+ep"
```

Comment:
Why remove that ? 
I suppose that it is because it is not needed for basic config listening on upper ports but maybe this need a comment to activate this line if gitea need to listen under 1024 port like ssh (port 22).

Reply:
@sapk added comment